### DISCUSSION TITLE
Granularity

### DISCUSSION THREAD

**Discussion by PhilippeDo** [cuebook/CueObserve#162] [General]
Why do we have only hour and daily granularity, I think Prophet is able to manage other granularity. have you planned to implement other granularity, like minute ?

**Comment by sachinkbansal** [comment]
@PhilippeDo We haven't yet tried Prophet with minute granularity. We'll add weekly granularity, but not in the immediate future.

If the minute granularity is important for you, would love to understand the use case. Have a few questions:
1. Would you schedule anomaly detection jobs to run every minute?
2. What type of input data are you looking at - system or business metrics?

**Reply by PhilippeDo** [reply]
Hi Sachinbansal, Thansk for your prompt response.

I'am currently working on use case related to systems metrics that comes from test performs regularly every one minute, 5 minutes, 10 minutes 15 minutes, 30 minutes and 60 minutes. 

So for each of these intervals I would like to forecast the next value. 

thanks
Best Regards

**Comment by sachinkbansal** [comment]
@PhilippeDo few more questions:

1. Do you only want to forecast? or do you want to detect anomalies as well?
2. Do you want to forecast only the next value or the next few values? e.g. for a 1-minute metric, get next 10 values as forecast.
3. What is your data source - a relational db or a timeseries db or something else?

**Reply by PhilippeDo** [reply]
1. We would like to do both, forecast and anomalies detection
2. For the moment forecast the next 3 values
3. I connected cueobserve with a mysql database

Thanks
